### PR TITLE
feat: seo keywords and robots.txt

### LINKED ISSUE #44: Improve SEO to score top search results
Verify that Plausible is tracking from desired stages and domains, and is using company credentials.
Verify that Sentry is tracking from desired stages and domains, and is using company credentials.
Verify the page is properly indexed by the search engines and sitemap is recognized.

### PR DESCRIPTION
**Pull request by lukeromanowicz**
## Description
I hope it resolves #44 but we will be able to find out once it's shipped to production.

## Demo
image
image

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] My change does not require a change to the documentation.

### CODE REVIEW

**Review by janmichek — APPROVED** [review]
Cool, it seems to be a promising solution. 👏  I guess the only way to find out is to deploy it on main domain

**Review by michele-franchi — COMMENTED** [review]
@lukeromanowicz  Did you also have a look at this comment about page titles consistency?
https://github.com/aeternity/aescan/issues/44#issuecomment-1592728681

Not sure if it's fixed.

**Comment by lukeromanowicz** [comment]
@michele-franchi Yes, it has been already fixed 2 months ago but released very recently in 0.3.0

File: src/utils/constants.js [hunk 1]
```diff
@@ -45,7 +45,8 @@ export const NUMBER_FRACTION_THRESHOLD = 100000
 export const MAXIMUM_FRACTION_DIGITS = 20
 export const APP_TITLE_SHORT = 'æScan'
 export const APP_TITLE = 'æScan - æternity Blockchain Explorer'
-export const APP_DESCRIPTION = 'æScan is a blockchain explorer, analytics platform, and decentralized Smart Contract navigation platform based on æternity'
+export const APP_DESCRIPTION = 'æScan is a blockchain explorer, analytics platform, and decentralized Smart Contract navigation platform based on æternity.'
+export const APP_KEYWORDS = 'aescan, aeternity, blockchain, explorer, search, crypto, currency'
```
Comment (michele-franchi) [inline src/utils/constants.js:49]: In my opinion I would remove "search, crypto, currency".

**Review by michele-franchi — APPROVED** [review]
I left a personal opinion but other than that it looks good!

File: src/utils/constants.js (hunk 1 above)
Comment (lukeromanowicz) [reply src/utils/constants.js:49]: I took that inspiration from etherscan and it seems to be working pretty well for them. @janmichek WDYT?
image

Comment (janmichek) [reply src/utils/constants.js:49]: My understanding of this topic is low, but the motivation behind choosing such keywords make complete sense to me

Comment (lukeromanowicz) [reply src/utils/constants.js:49]: Since we would be left with only 4 keywords which are well within the usual number of keywords, I'll keep them then.

**Pull request merged**